packages/cli/src/commands/generate/cell/cell.js

```diff
@@ -66,22 +67,23 @@ export const files = async ({
     (isWordPluralizable(cellName) ? isPlural(cellName) : options.list) ||
     options.list
 
+  // needed for the singular cell GQL query find by id case
+  try {
+    model = await getSchema(pascalcase(singularize(cellName)))
+    idType = getIdType(model)
+    mockIdValue = idType === 'String' ? "'42'" : 42
+  } catch {
+    // eat error so that the destroy cell generator doesn't raise when try to find prisma query engine in test runs
+    // assume id will be Int, otherwise generated will keep throwing
```

Comment:
"generated" what? Generated code? Generated cell? Generated something else?

Reply:
Good question. I didn't update the comments, but this doesn't make sense. I'll re-read and review.